README.md

```diff
@@ -96,6 +97,9 @@ A categorized community-driven collection of awesome Pharo libraries, tools, fra
 + [Stylesheet](https://github.com/pharo-contributions/Stylesheet) - Stylesheet is a project to define css like stylesheet in Pharo applications.
 + [Units](https://github.com/zweidenker/Units) - A simple package for Units management in Pharo.
 
+## Network protocols
++ [JRPC](https://github.com/juliendelplanque/JRPC) - Yet another [JSON-RPC 2.0](https://www.jsonrpc.org/specification) implementation for Pharo Smalltalk.
```

Comment:
Are there others?

I only found yours by looking fast.

Reply:
Probably, but let's people add them themselves. Or we will add them later...

Reply:
Oh, you mean JRPC implementations? yes, see the table at bottom of the readme of this repo.